src/mono/netcore/sample/wasm/Directory.Build.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <PropertyGroup>
+    <Configuration>Release</Configuration>
+    <OutputType>Exe</OutputType>
+
+    <RunAOTCompilation Condition="'$(RunAOTCompilation)' == ''">false</RunAOTCompilation>
+    <PublishTrimmed>true</PublishTrimmed>
+    <TrimMode>link</TrimMode>
+    <RunAnalyzers>false</RunAnalyzers>
+  </PropertyGroup>
+
+  <Import Project="..\Directory.Build.props"/>
```

Comment:
Move to top?

Reply:
Oops, missed this comment here:
`<!-- These need to be set here because the root Directory.Build.props sets up the intermediate path early -->`